src/content/getting-started/exposing-workloads/index.md

```diff
@@ -117,73 +118,13 @@ Once the ingress is up, you will be able to access your service publicly at a UR
 
-## Forwarding an authenticated port with `kubectl port-forward` {#port-forward}
-
-Forwarding a port with `kubectl` is fairly easy, however, it only works with single pods and not with services. Thus you need the exact pod name. You can either get this manually by running
+## Forwarding ports with `kubectl port-forward` {#port-forward}
 
-```nohighlight
-kubectl -n NAMESPACE get pods
-```
-
-and looking for the right pod name. Or by running following script.
-
-```nohighlight
-POD=$(kubectl get pods -n NAMESPACE --selector <label-key>=<label-value> \
-    -o template --template '{{range .items}}{{.metadata.name}} {{.status.phase}}{{"\n"}}{{end}}' \
-    | grep Running | head -1 | cut -f1 -d' ')
-```
+Forwarding a port with `kubectl` is fairly easy, however, it should be only used for debugging purposes.
 
-Be sure to have your pod labeled accordingly so you can find it with the above selector.
-
-After this you can run
-
-```nohighlight
-kubectl port-forward -n NAMESPACE ${POD} <local-port>:<pod-port>
-```
-
-or to have it running in the background
-
-```nohighlight
-kubectl port-forward -n NAMESPACE $POD <local-port>:<pod-port> &
-```
-
-Now you can access your pod locally via `localhost:<local-port>`.
-
-## Access any service from through the API proxy {#api-access}
-
-The Kubernetes API comes with an inbuilt proxy, which you can use to access services deployed on your cluster. The URL schema is
-
-```nohighlight
-https://api.CLUSTER_ID.k8s.gigantic.io/api/v1/proxy/namespaces/NAMESPACE/services/SERVICE_NAME:PORT_NAME/proxy/
-```
-
-If the service does not use a named port, `PORT_NAME` must be the port number.
-
-Access will only be granted to clients which
-
-- trust the API's server certificate, which means they trust the Certificate Authority (CA) that signed it and
-- provide a valid client certificate.
-
-The Giant Swarm [web user interface]({{< relref "/ui-api/web/" >}}) shows you how to obtain the certificate files.
-
-To make these certificates available to HTTP clients/browsers, see our guide [Establishing Trust to Your Cluster's CA and Importing Certificates]({{< relref "/getting-started/ca-certificate" >}}) which explains this for different clients on various platforms.
-
-### Example
-
-Assuming you have a service `elasticsearch` with a port named `es` in the `logging` namespace, you could
-access the Elasticsearch index stats API like this:
-
-```nohighlight
-curl -v -u username:password \
-  --cacert ./ca.pem \
-  --cert ./crt.pem \
-  --key ./key.pem \
-  https://api.CLUSTER_ID.k8s.gigantic.io/api/v1/namespaces/logging/services/elasticsearch:es/proxy/_stats
-```
+To know more about forwarding TCP ports using `kubectl port-forward`, you should read the ["Use Port Forwarding to Access Applications in a Cluster"](https://kubernetes.io/docs/tasks/access-application-cluster/port-forward-access-application-cluster/) page in the official Kubernetes documentation.
```

Comment:
why do you remove the alternative ways? it is not longer possible to use the proxy?

Reply:
Check out [this discussion](https://github.com/giantswarm/docs/pull/1109#discussion_r700964202). We decided to make the guide Ingress focussed, while the other ways are still valid, Ingress seems to be the canocical way to expose your services. The other methods are also explained upstream.